src/main/java/pages/HomePage.java

```diff
@@ -59,4 +62,13 @@ public boolean isVisibleMyArticlesLink(){
     public boolean isVisibleFavoritedArticles(){
         return elementIsDisplayed(lnkFavoritedArticles);
     }
+
+    public void clicEditProfileSettingsLink(){
+        clickElement(lnkEditProfileSettings);
+    }
+
+    public String getUserNameLink(){
+        forceWait();
```

Comment:
Is it necessary to add this implicit wait?

Reply:
this wait was put so that you can load or update the new username, but I will create a method so that it expects an element where the attribute has a certain value
 `public void waitForAttributeValue(int timeout, WebElement element, String attribute, String value){
        waitForCondition( wd->element.getAttribute(attribute).equals(value),timeout );
    }`